### DISCUSSION TITLE
Window switcher action

### DISCUSSION THREAD

**Discussion by andrewsardone** [Ptujec/LaunchBar#11] [Ideas]
Does anyone have a similar LaunchBar action to [mandrigin/AlfredSwitchWindows](https://github.com/mandrigin/AlfredSwitchWindows)? Or an option for searching active-application/all-open windows and selecting within LaunchBar?

**Comment by Ptujec** [comment]
Intriguing idea, Andrew! You can press and hold cmd and then press space repeatedly to scroll through applications. From there you can access recent documents with arrow keys. But it is not exactly the same. I usually use the system shortcuts to switch between apps and windows of the current app. However an action listing all open windows (maybe even tabs) sounds like a cool thing I would definitely try.

**Reply by andrewsardone** [reply]
Agreed. I also routinely change apps via <kbd>⌘</kbd> + <kbd>tab</kbd> or LaunchBar, and then cycle the app's windows with <kbd>⌘</kbd> + <kbd>`</kbd>. But when I have a lot of tabs and windows open, I could see using the power of LaunchBar's fuzzy searching to quickly switch to, say, a known open document.

Recently I was playing with [Contexts.app](https://contexts.co/) and enjoying its [window-search feature][ws]. Seems like a natural action extension for LaunchBar.

[ws]: https://cdn.contexts.co/static/website/media/feature--search.mp4
